### PR TITLE
#46 Remediation/Mitigation

### PR DESCRIPTION
**Pull request by laurie-tyz**
Shifted two paragraphs up in 040_trees....  This change may result in the need for a new subtitle and line 14, "The next two sections...."

### CODE REVIEW

File: doc/version_1/040_treesForVulMgmt.md [hunk 1]
```diff
@@ -22,6 +22,11 @@ Relatedly, C-level executives and public policy professionals often make, shape,
...
 
+<<new subtitle here?>>
```
Comment (j---) [inline doc/version_1/040_treesForVulMgmt.md:25]: How about "Vulnerability Management Actions"?

**Review by j--- — COMMENTED** [review]
Besides the new subtitle, looks good. But we should get that settled now.

File: doc/version_1/040_treesForVulMgmt.md (hunk 1 above)
Comment (laurie-tyz) [reply doc/version_1/040_treesForVulMgmt.md:25]: I plan to add more details under this section to answer #79 and make the links to the decisions that are influenced by remediation. 
What about "Enumerating Vulnerability Management Actions"?

Comment (j---) [reply doc/version_1/040_treesForVulMgmt.md:25]: Works for me.

**Comment by laurie-tyz** [comment]
Creating a new pull request with the heading

**Pull request closed without merging**